docs/specs/toc/toc.yml

```diff
@@ -1522,3 +1522,16 @@ outputs:
         "a.md": [{ "source": "TOC.yml", "type": "metadata" }]
       }
     }
+---
+# TOC with output-type="json"
+inputs:
+  docfx.yml: |
+    outputType: Json
+  TOC.yml: |
+    - name: a
+      href: a.md
+  a.md:
+outputs:
+  a.json:
+  toc.json: |
```

Comment:
Also need a test in `static.yml` to cover `outputType=html` + `toc.json.js`

Reply:
I add a mustache template file with a js file to generate `.html` and `.json` outputs.